tools/src/main/java/org/apache/kafka/tools/StreamsResetter.java

```diff
@@ -653,6 +650,7 @@ public StreamsResetterOptions(String[] args) {
                 if (CommandLineUtils.isPrintVersionNeeded(this)) {
                     CommandLineUtils.printVersionAndExit();
                 }
+                CommandLineUtils.checkRequiredArgs(parser, options, bootstrapServerOption);
```

Comment:
Could you also add a test to verify that the `bootstrap-server` argument is required and that the command fails with a proper error message if it is missing?